ui/sections/network/controls/table-routes.reel/table-routes.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @module ui/table-routes.reel
+ */
+var AbstractMultipleEditController = require('ui/abstract/abstract-multiple-edit-controller').AbstractMultipleEditController,
+    Model = require("core/model/model").Model;
+
+/**
+ * @class TableRoutes
+ * @extends Component
+ */
+exports.TableRoutes = AbstractMultipleEditController.specialize({
+
+    getModelType: {
+        value: function() {
+            return Model.NetworkRoute;
+        }
+    },
+
+    loadObjects: {
+        value: function() {
+            return this._sectionService.loadStaticRoutes();
+        }
+    },
+
+    saveObject: {
+        value: function(object) {
+            return this._sectionService.saveStaticRoute(object);
+        }
+    },
+
+    deleteObject: {
+        value: function(object) {
+            return this._sectionService.deleteStaticRoute(object);
+        }
+    },
+
+    mapObjectToRawData: {
```

Comment:
Raw data notion must not be known to the component. This kind of mapping needs to be done only by the ORM (in our case, montage-data).